R/cli.R

```diff
@@ -78,24 +76,35 @@ format_info_basic <- function(internal) {
   asymmetric <- internal$parameters$asymmetric
   confounding <- internal$parameters$confounding
   testing <- internal$parameters$testing
+  group <- internal$parameters$group
+  group_lags <- internal$parameters$group_lags
 
   feat_group_txt <- ifelse(is_groupwise, "group-wise", "feature-wise")
-  iterative_txt <- ifelse(iterative, "iterative", "non-iterative")
+  iterative_txt <- ifelse(iterative, "Iterative", "Non-iterative")
```

Comment:
Since the other information does not start with a capital leter.

```suggestion
  iterative_txt <- ifelse(iterative, "iterative", "non-iterative")
```